moz-webgpu-cts/src/shared.rs

```diff
@@ -788,4 +884,16 @@ fn runner_url_path() {
         .to_string(),
         "_mozilla/blarg/stuff.https.html?win",
     );
+
+    assert_eq!(
+        TestPath::from_metadata_test(
+            Browser::Servo,
+            Path::new("tests/wpt/webgpu/meta/webgpu/cts.https.html.ini"),
+            "cts.https.html?win"
+        )
+        .unwrap()
+        .runner_url_path()
+        .to_string(),
+        "_webgpu/webgpu/cts.https.html?win",
```

Comment:
issue: This is actually wrong, and it took me a while to figure out why! We weren't preserving the `_webgpu/` prefix on output here, and that's supposed to be here (viz., to mirror how we parse `_webgpu/` in URL path parsing logic).

Pushed a `fixup!` with 9027a2f.

Reply:
FTR: Finding this actually revealed that we don't test a round-trip between URL path parsing and emission for `TestPath`s, and that made me write <https://github.com/ErichDonGubler/moz-webgpu-cts/pull/101>.